ui/src/app/base/hooks.ts

```diff
@@ -257,6 +257,26 @@ export const useVisible = (
   return [value, toggleValue];
 };
 
+/**
+ * Track and untrack ids.
+ */
+export const useTrackById = (): {
```

Comment:
Is it possible use a generic here? Just because in a few places the ids are strings instead of numbers.
``` typescript
export const useTrackById = <T>(): {
  tracked: T[];
  toggleTracked: (id: T) => void;
} => {
...etc
```